apps/console/src/features/userstores/components/wizards/general-details-userstore.tsx

```diff
@@ -220,6 +241,30 @@ export const GeneralDetailsUserstore: FunctionComponent<GeneralDetailsUserstoreP
                                 "description.placeholder") }
                             value={ values?.get("description")?.toString() }
                             data-testid={ `${ testId }-form-description-textarea` }
+                            validation={ async (value: string, validation: Validation) => {
+
+                                const regExpInvalidSymbols: RegExp = new RegExp(
+                                    USERSTORE_VALIDATION_REGEX_PATTERNS.xssEscapeRegEx);
+
+                                let isMatch: boolean = false;
+                                let invalidStringValue: string = null;
+
+                                if (regExpInvalidSymbols.test(value)) {
+                                    isMatch = true;
+                                    invalidStringValue = regExpInvalidSymbols.exec(value).toString();
+                                }
```

Comment:
Better to move it to a seperate function because this code is duplicated in the other field as well.

ex:

```
const checkValidityOfAddUserstoreFields = (value: string): Map<string, string|boolean>() => {
  const regExpInvalidSymbols: RegExp = new RegExp( 
     USERSTORE_VALIDATION_REGEX_PATTERNS.xssEscapeRegEx);

     let isMatch: boolean = false;
     let invalidStringValue: string = null;

     if (regExpInvalidSymbols.test(value)) {
         isMatch = true;
         invalidStringValue = regExpInvalidSymbols.exec(value).toString();
     }

    const validityResultsMap: Map<string, string|boolean> = new Map<string, string|boolean>();
    validityResultsMap.set("isMatch", isMatch);
    validityResultsMap.set("invalidStringValue", invalidStringValue);

   return validityResultsMap;
}
```